src/Google.Cloud.Functions.Invoker/Logging/LoggerBase.cs

```diff
@@ -39,6 +46,12 @@ internal abstract class LoggerBase : ILogger
         /// </summary>
         public void Log<TState>(LogLevel logLevel, EventId eventId, TState state, Exception exception, Func<TState, Exception, string> formatter)
         {
+            // Functions expect to go for a long time without any CPU. It's reasonable to suppress this warning.
+            if (_isKestrelCategory && eventId.Id == HeartbeatSlowEventId)
```

Comment:
Would it be possible to make this configurable? Even if we suppress it by default...
My heart is not set on it though.

Reply:
Getting configuration to the logger would be fairly tortuous, I think. Let's leave it as unconfigurable for now, and wait for the feature request :)